src/api/c/fft.cpp

```diff
@@ -247,3 +247,14 @@ af_err af_fft3_c2r(af_array *out, const af_array in, const double norm_factor, c
 {
     return fft_c2r<3>(out, in, norm_factor, is_odd);
 }
+
+af_err af_set_fft_plan_cache_size(size_t cache_size)
+{
+#ifndef AF_CPU
```

Comment:
Don't do this. Just have a dummy inline function in CPU backend's fft.hpp. The compiler will optimize it out.